.github/workflows/label.yml

```diff
@@ -0,0 +1,23 @@
+name: Labeller
+on:
+  pull_request_target:
+    types: closed
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: .github/scripts
+      - name: Add release labels on merge
+        run: |
+          .github/scripts/pr-label-issues.sh "${{ github.event.pull_request.number }}" "$GITHUB_REPOSITORY" "$GITHUB_BASE_REF" >> "$GITHUB_STEP_SUMMARY"
+        if: github.repository	== 'infinispan/infinispan' && github.event_name == 'pull_request_target' && github.event.action == 'closed' && github.event.pull_request.merged == true
```

Comment:
I think we can simplify this to:

```yaml
 github.repository == 'infinispan/infinispan' && github.event.pull_request.merged == true
Comment
```

As we're only executing this workflow when we have pull_request_target and type==closed.

Reply:
Done